go/test/endtoend/vreplication/resharding_workflows_v2_test.go

```diff
@@ -170,6 +174,72 @@ func tstWorkflowExec(t *testing.T, cells, workflow, sourceKs, targetKs, tables,
 	return nil
 }
 
+// tstWorkflowExecVtctl executes a MoveTables or Reshard workflow command using
+// vtctlclient. It should operate exactly the same way as tstWorkflowExec, but
+// using the legacy client.
+func tstWorkflowExecVtctl(t *testing.T, cells, workflow, sourceKs, targetKs, tables, action, tabletTypes,
```

Comment:
This is a copy of what `tstWorkflowExec() was prior to https://github.com/vitessio/vitess/pull/15579